### PR TITLE
Improved  BlameControlTests

### PR DESCRIPTION
**Pull request by RussKie**

### CODE REVIEW

File: UnitTests/GitUITests/UserControls/BlameControlTests.cs [hunk 1]
```diff
@@ -5,13 +5,14 @@
...
 using FluentAssertions;
 using GitCommands;
-using GitUI;
 using GitUI.Blame;
 using GitUIPluginInterfaces;
 using NUnit.Framework;
 
 namespace GitUITests.UserControls
 {
+    [SetCulture("en-US")]
```
Comment (pmiossec) [inline UnitTests/GitUITests/UserControls/BlameControlTests.cs]: I like that (a lot more than the deleted lines just below)!

But I'm really not fond of the "en-US" culture chosen (rant: I really dislike their dumb and only used by them date format) but I'm not the center of the universe 😉.

File: UnitTests/GitUITests/UserControls/BlameControlTests.cs [hunk 2]
```diff
@@ -22,25 +23,15 @@ public class BlameControlTests
...
         {
-            CultureInfo.CurrentCulture = new CultureInfo("EN-GB");
-            var blameControlTask = ThreadHelper.JoinableTaskContext.Factory.RunAsync(async () =>
-            {
-                await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
-                return new BlameControl();
-            });
-
-            var blameControl = blameControlTask.Join();
-            _sut = new BlameControl.TestAccessor(blameControl);
```
Comment (pmiossec) [inline UnitTests/GitUITests/UserControls/BlameControlTests.cs]: 👍

File: UnitTests/GitUITests/UserControls/BlameControlTests.cs [hunk 3]
```diff
@@ -68,68 +61,85 @@ public void SetUp()
...
+                gutterLines[1].Should().NotBeNullOrEmpty().And.BeNullOrWhiteSpace();
+                gutterLines[2].TrimEnd().Should().Be("3/22/2011 12:01 PM - author2");
+                gutterLines[3].Should().NotBeNullOrEmpty().And.BeNullOrWhiteSpace();
+
+                // All the lines have the same length
+                gutterLines.Select(l => l.Length).Should().AllBeEquivalentTo(gutterLines[0].Length);
+            }
+            finally
+            {
+                AppSettings.BlameShowAuthorTime = originalValue;
```
Comment (pmiossec) [inline UnitTests/GitUITests/UserControls/BlameControlTests.cs]: restoring initial value 👍

**Comment by RussKie** [comment]
Thank you!
I saw it failed but didn't have time to fix it.

I don't like en-US locale too, I much prefer en-AU or en-GB myself :)
I changed to en-US because en-GB was failing for me for some strange
reason... It was late and I was tired to debug it.
In all other tests we use en-US too.

**Comment by pmiossec** [comment]
>I don't like en-US locale too, I much prefer en-AU or en-GB myself :) 

&

> In all other tests we use en-US too.

I supposed it was the reason but some rant make you sometimes feel better 🤣

**Comment by RussKie** [comment]
😃

**Pull request merged**